ios/RecordingSession.swift

```diff
@@ -193,15 +194,22 @@ class RecordingSession {
   func finish() {
     ReactLogger.log(level: .info, message: "Finishing Recording with AssetWriter status \"\(assetWriter.status.descriptor)\"...")
 
+    if(hasRunningWritingAttempt) {
+      ReactLogger.log(level: .info, message: "Detected two finish() calls on the same AVWriter single-use object, returning to prevent crash")
```

Comment:
Make error message say `"Tried calling `finish()` twice while AssetWriter is still writing!"` and make it a `.warning` or `.error`